rn/Teacher/src/modules/courses/settings/CourseSettings.js

```diff
@@ -227,6 +228,12 @@ const styles = createStyleSheet((colors, vars) => ({
     color: colors.textDarkest,
     lineHeight: 54,
   },
+  inputText: {
+    flex: 3,
+    color: colors.textDarkest,
+    fontSize: 16,
+    textAlign: 'right',
```

Comment:
the `lineHeight` property was making the input ridiculously big & the text was clipped below the visible space. This works around the awful bug in RN.